core/src/com/unciv/logic/automation/civilization/MotivationToAttackAutomation.kt

```diff
@@ -105,7 +104,7 @@ object MotivationToAttackAutomation {
         }
 
         if (diplomacyManager.resourcesFromTrade().any { it.amount > 0 })
-            modifiers.add(Pair("Receiving trade resources", -8f * personality.modifierFocus(PersonalityValue.Commerce, .5f) * personality.modifierFocus(PersonalityValue.Loyal, .2f)))
+            modifiers.add(Pair("Receiving trade resources", -8f * personality.modifierFocus(PersonalityValue.Commerce, .5f) * personality.modifierFocus(PersonalityValue.Military, .2f)))
```

Comment:
Why military? Because strategic resources? There's also luxuries!
This should be something that's disconnected from personality IMO

Reply:
High Military will increase the amount of military units produced, which increase the Force advantage over the opponent, and as such make the AI indirectly more likely to declare war. Instead of then declaring war on just anyone, imo it should take into account the trades more, and stay at peace with those that provide strategic (necessary for building strong military units) and other resources.